superflore/generators/bitbake/run.py

```diff
@@ -78,6 +78,7 @@ def main():
         selected_targets = get_distros_by_status('active')
     repo_org = 'allenh1'
     repo_name = 'meta-ros'
+    branch_name = args.upstream_branch or ''
```

Comment:
I don't think you need the `or ''` at the end of this line, do you?

Reply:
Removed, thanks!